control-plane/pkg/keda/keda.go

```diff
@@ -204,3 +204,25 @@ func addAuthSecretTargetRef(parameter string, secretKeyRef v1beta1.SecretValueFr
 	secretTargetRefs = append(secretTargetRefs, ref)
 	return secretTargetRefs
 }
+
+func SetAutoscalingAnnotations(objannotations map[string]string) map[string]string {
+	if objannotations != nil {
+		cgannotations := map[string]string{}
+		setAnnotation(objannotations, keda.AutoscalingClassAnnotation, cgannotations)
+		setAnnotation(objannotations, keda.AutoscalingMinScaleAnnotation, cgannotations)
+		setAnnotation(objannotations, keda.AutoscalingMaxScaleAnnotation, cgannotations)
+		setAnnotation(objannotations, keda.KedaAutoscalingPollingIntervalAnnotation, cgannotations)
+		setAnnotation(objannotations, keda.KedaAutoscalingCooldownPeriodAnnotation, cgannotations)
+		setAnnotation(objannotations, keda.KedaAutoscalingKafkaLagThreshold, cgannotations)
```

Comment:
Just FYI, a new setting: `activationLagThreshold` has been introduced in KEDA 2.8.1, adding to `eventing-autoscaler-keda` here: https://github.com/knative-sandbox/eventing-autoscaler-keda/pull/329

Reply:
Ah thanks @zroubalik for the 2.8.1 updates, and review. I will add this in a forthcoming PR too.

Reply:
#2639